docs/docs/01-quickstart.md

```diff
@@ -302,11 +302,13 @@ Refocus lets you limit the number of requests per user--once the user reaches th
 Configure your limits using the following environment variables:
 For more details, see the [express-limiter](https://www.npmjs.com/package/express-limiter) module.
 
-- `EXPRESS_LIMITER_PATH` - the URI to limit; allows wildcards; use "*" to limit  all requests
-- `EXPRESS_LIMITER_METHOD` - the HTTP method to limit; use "all" (in lower case) to limit all HTTP methods
+- `EXPRESS_LIMITER_PATH` - the URI to limit. Can be any valid [express path](http://expressjs.com/en/4x/api.html#path-examples). To specify multiple paths, separate them with commas, e.g. "/v1/aspects,/v1/subjects". Use "*" to limit  all requests
```

Comment:
so the limiter works like this?
EXPRESS_LIMITER_PATH  = "url1, url2"
EXPRESS_LIMITER_METHOD = "post,put"
would limit url1's POST and PUT, and url2's POST and PUT?

what if the user wants to limit only the POST of url1, and the PUT of url2?

Reply:
what if the user wants to POST url1, PUT url2, and PATCH url3? Does the design allow for that?

Reply:
Good point. No, that's not possible with the current design.

The configuration could definitely be a lot more flexible. Currently you also can't set different configs for different paths and methods, or have an arbitrary number of limiter configs. We could do that pretty easily, but we'd have to figure out the best way to do all the environment variables.

The purpose of this story was to allow setting up two different configurations for the same path/method so we could restrict it to e.g. no more than 60 requests per hour and also no more than 1 request per 100ms. I think more complicated configuration is outside the scope of this story. The change to allow multiple paths and methods wasn't even really necessary, I just added that because I was already doing it for the lookup. 

Is this something that we need? Should we create a different story for it?

@iamigo